sufia-models/lib/sufia/models/stats/user_stat_importer.rb

```diff
@@ -47,6 +51,22 @@ def delay
       sleep @delay_secs
     end
 
+    def rescue_and_retry fail_message
```

Comment:
Include parens for consistency with the rest of the methods defined in this file?